articles/ai-services/content-safety/includes/env-vars.md

```diff
@@ -21,29 +21,29 @@ In this example, you'll write your credentials to environment variables on the l
 
 To set the environment variable for your key and endpoint, open a console window and follow the instructions for your operating system and development environment.
 
-1. To set the `CONTENT_SAFETY_KEY` environment variable, replace `your-key` with one of the keys for your resource.
-2. To set the `CONTENT_SAFETY_ENDPOINT` environment variable, replace `your-endpoint` with the endpoint for your resource.
+1. To set the `CONTENT_SAFETY_KEY` environment variable, replace `YOUR CONTENT SAFETY KEY` with one of the keys for your resource.
+2. To set the `CONTENT_SAFETY_ENDPOINT` environment variable, replace `YOUR CONTENT SAFETY ENDPOINT` with the endpoint for your resource.
 
 #### [Windows](#tab/windows)
 
 ```console
-setx CONTENT_SAFETY_KEY your-key
+setx CONTENT_SAFETY_KEY 'REPLACE WITH YOUR CONTENT SAFETY KEY'
```

Comment:
The name here needs to match the name in the instructions. Same for the other commands